features/steps/formatting_steps.rb

```diff
@@ -15,6 +15,10 @@
   add_run_input SAMPLE_COMPILE_STORYBOARD
 end
 
+Given(/^I have an asset catalog to compile$/) do
```

Comment:
I haven't notice this file when I added paring in the past. What is it used for? Don't you need a `Then` block after the `Given` block?

Reply:
It's used in the `.feature` files. I added a test for it now.